src/utils/types.ts

```diff
@@ -148,6 +150,8 @@ interface ResponsesByCode {
   [key: string]: ResponseFull | null
 }
 
+type ReviewDecision = 'Approve' | 'Reject'
```

Comment:
Should these be defined by database enum?

Reply:
Yes. I've changed that on the next PR: #221.